test/test_helper.rb

```diff
@@ -30,6 +30,9 @@
 
 require "webmock/minitest"
 
+# Silence log output
```

Comment:
Is there a better way to do this?

Without it, the minitest output is interrupted by log output from the client. Minitest doesn't have any built in options to suppress stdout from code under test.